frontend/src/components/headerSection/header.css

```diff
@@ -1,5 +1,6 @@
 .app-header {
-    min-height: 70vh;
```

Comment:
Also, the reason I want to use vh is because of something like this
![image](https://user-images.githubusercontent.com/6146988/112249216-5dc80400-8c25-11eb-8142-7cacf8fc7861.png)

Kinda looks like a broken page with no cue to go down.

Reply:
:/

Reply:
Another sadness
![image](https://user-images.githubusercontent.com/6146988/112263002-890a1d80-8c3c-11eb-917f-0fb258dc1e05.png)